frontend/src/components/UserProfiles/ProfileListings.js

```diff
@@ -9,27 +9,17 @@ const ProfileListings = ({ rides, handleCancel }) => {
     const activeRides = rides.filter((ride) => !ride.status || ride.status === 0);
     const cancelledRides = rides.filter((ride) => ride.status && ride.status === 2);
     const activeRideList = activeRides.map(
-        ({ startLoc, endLoc, time, price, capacity, _id }) => (
+        (ride) => (
```

Comment:
I think .map function requires a key associated with each ride to be passed into the param

Reply:
> I think .map function requires a key associated with each ride to be passed into the param

Not sure what do you mean, but are you talking about the warnings that appears in Chorme DevTools' console?
`Warning: Each child in a list should have a unique "key" prop.` If that's the problem, I can quickly fix that. If not, could you elaborate more on the problem here?

Reply:
@EvelynL09 yes, that is the problem I was concern about.